airflow/www/security.py

```diff
@@ -360,8 +365,9 @@ def merge_pv(perm, view_menu):
             .filter(or_(models.DagModel.is_active, models.DagModel.is_paused))\
             .filter(~models.DagModel.is_subdag).all()
```

Comment:
1. Here you're fetching all DAGs which is **paused** & non-subdag. But in the docstring above, you mentioned "fetching all the existing dag models that are **non-paused** or non subdags.".
2. In addition, the relations here is `OR(is_active, is_paused) AND non-subdag`. But in the docstring it is simply `non-paused OR non-subdags`

Conflicting to me. Did I miss something?